### ISSUE TITLE
no command for selector: noop:

### ISSUE THREAD

**Issue by firedev** [eddorre/SublimeERB#19]
It works on one computer, but not on a another, I add the shortcut without context, but anyway. I press the shortcut and nothing happens. Seen this before, can't figure out what might causing it.

Here is what I see in console:

```
no command for selector: noop:
```

Any ideas ?

**Comment by eddorre** [comment]
@firedev,

I have a few questions:
1. Are the two computers running the same OS?
2. Did you install via source or package control?
3. What's in your user keybinding?

**Comment by firedev** [comment]
I think this is because the shortcut should be encased in the square brackets like this:

[
  { "keys": ["ctrl+shift+."], "command": "erb" }
]

and in the beginning the user keybindings file is empty and there are no square brackets. This can be closed I think.

**Comment by cschep** [comment]
If you run into this like I just did.. try installing the package again. I think the install failed in the background. I finally resolved to remove the plugin and try reinstalling, but it didn't show up in my remove list. Odd.

Went and installed it .. "again". Worked fine.

I think noop issue means that ST2 can't find the plugin, nothing wrong with the keyboard shortcut syntax.

Anyway, reinstall fixed it for me! Thanks!
